From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: [PATCH] qtdbus needs qtscript

When building qt without QtScript support, I get this error:

make[4]: Entering directory `qt-embedded-linux-opensource-src-4.5.0/src/plugins/script/qtdbus'
rm -f libqtscriptdbus.so
i586-unknown-linux-gnu-g++ -L${sysroot}/lib -L${sysroot}/usr/lib -Wl,-rpath-link -Wl,${sysroot}/lib -Wl,-rpath-link -Wl,${sysroot}/usr/lib -Wl,-rpath,/usr/lib -Wl,-rpath,/usr/lib -shared -o libqtscriptdbus.so .obj/release-shared-emb-i386/main.o .obj/release-shared-emb-i386/moc_main.o -Lqt-embedded-linux-opensource-src-4.5.0/lib -L${sysroot}/lib -L${sysroot}/usr/lib -lQtDBus -Lqt-embedded-linux-opensource-src-4.5.0/lib -L${sysroot}/lib -L${sysroot}/usr/lib -lQtXml -lQtScript -lQtGui -lts -lpng -lfreetype -lQtNetwork -lQtCore -lz -lm -lrt -lpthread -ldl
${TOOLCHAIN}/bin/../lib/gcc/i586-unknown-linux-gnu/4.3.2/../../../../i586-unknown-linux-gnu/bin/ld:
cannot find -lQtScript

The qtdbus directory needs qtscript, so we reflect this in script.pro.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 src/plugins/script/script.pro |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: qt-embedded-linux-opensource-src-4.5.0/src/plugins/script/script.pro
===================================================================
--- qt-embedded-linux-opensource-src-4.5.0.orig/src/plugins/script/script.pro
+++ qt-embedded-linux-opensource-src-4.5.0/src/plugins/script/script.pro
@@ -1,2 +1,2 @@
 TEMPLATE = subdirs
-contains(QT_CONFIG, dbus):SUBDIRS += qtdbus
+contains(QT_CONFIG, dbus):!isEmpty(QT_BUILD_SCRIPT_LIB):SUBDIRS += qtdbus
